specification/dataprotection/resource-manager/Microsoft.DataProtection/preview/2021-02-01-preview/dataprotection.json

```diff
@@ -1520,6 +1520,345 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DataProtection/backupVaults/{vaultName}/backupInstances/{backupInstanceName}/resumeBackups": {
+      "post": {
```

Comment:
>post [](http://example.com/codeflow?start=7&length=4)

Should add a description for each of the five new operations.

Reply:
Added.. Thanks.